libs/server-sdk/README.md

```diff
@@ -102,6 +102,34 @@ for consistency across SDKs, as well as test networking behavior in a long-runni
 method in the SDK, and verify that event sending, flag evaluation, stream reconnection, and other aspects of the SDK all
 behave correctly.
 
+Validating SDK packages with the SLSA framework
+-------
+
```

Comment:
I'd add a note that this does not apply if you're building the SDK from source (unless our dependencies one day implement SLSA.)

Reply:
Thanks Casey! Will add that note today